pkg/rbeconfigsgen/rbeconfigsgen.go

```diff
@@ -648,19 +648,21 @@ func processTempDir(o *Options) error {
 	return nil
 }
 
+func genCppToolchainTarget(o *Options) string {
+	if o.OutputConfigPath != "" {
+		return fmt.Sprintf("//%s/cc:%s", path.Clean(o.OutputConfigPath), o.CPPToolchainTargetBasename)
```

Comment:
Add a string replace for "\\" -> "/" in o.OutputConfigPath and unit test for this case. I don't think Bazel allows "\\" in labels. Also, ensure you don't mutate `o.OutputConfigPath`, so something like:

```
configPath := string.Replace(o.OutputConfigPath, "\\", "/")
if configPath != "" {
...
```

Reply:
Done. Also added some checks taht OutputConfigPath isn't an absolute path on Validation.